plugins/woocommerce-admin/client/marketing/hooks/test/useCampaigns.test.ts

```diff
@@ -0,0 +1,230 @@
+/**
+ * External dependencies
+ */
+import { renderHook } from '@testing-library/react-hooks/dom';
+import { waitFor } from '@testing-library/react';
+import { dispatch } from '@wordpress/data';
+import 'whatwg-fetch'; /* eslint-disable-line import/no-unresolved */ /* To make sure Response is available */
+
+/**
+ * Internal dependencies
+ */
+import { useCampaigns } from '../useCampaigns';
+import { useRegisteredChannels } from '../useRegisteredChannels';
+import { STORE_KEY } from '../../data-multichannel/constants';
+import { RegisteredChannel } from '../../types';
+import { Campaign as APICampaign } from '../../data-multichannel/types';
+import '../../data-multichannel'; // To ensure the store is registered
+
+type Channel = Pick< RegisteredChannel, 'slug' | 'title' | 'icon' >;
```

Comment:
Comparing the single line of the type definition to the given example, this will require more lines of code and looks redundant when calling `mockRegisteredChannels`. I lean toward keeping it as it is.